src/main/kotlin/io/snyk/plugin/Severity.kt

```diff
@@ -0,0 +1,39 @@
+package io.snyk.plugin
+
+import com.intellij.util.ui.UIUtil
+import java.awt.Color
+
+class Severity {
+    companion object {
+        const val CRITICAL = "critical"
+        const val HIGH = "high"
+        const val MEDIUM = "medium"
+        const val LOW = "low"
+
+        fun getIndex(severity: String): Int =
+            when (severity) {
+                CRITICAL -> 4
+                HIGH -> 3
+                MEDIUM -> 2
+                LOW -> 1
+                else -> 0
+            }
+
+        fun toName(index: Int): String =
+            when (index) {
```

Comment:
CRITICAL is foggoten here

Reply:
@ArtsiomCh I don't added it yet here because as I understand critical severity not implemented for Snyk code for now. And this method used in Snyk code source code part. So just afraid to broke some Snyk code logic. But if you confident that this not broke anything it will be good to add. Thanks! Good catch! 👍

Reply:
AFAIK there been some rumors about Critical severity implementation in Snyk Code already, but I'm not sure.
Anyway, here it's just an number, so if we never get severity with index 4 from Snyk Code, then we just not show it :)
So adding `4 -> CRITICAL` is safe and should prevent brakeage later when Snyk Code actually will start giving us critical issues.